scripts/test/acceptance/default.nix

```diff
@@ -100,4 +100,57 @@ in
     echo "Blockchain sync complete!"
     echo "Blockchain height: $height blocks"
     echo "Elapsed time: $elapsed_time seconds"
+
+    # Restore a wallet
+    echo
+    echo "Going to restore a wallet"
+    curl --silent --cacert ${stateDir}/tls/client/ca.crt --cert ${stateDir}/tls/client/client.pem -H "Content-Type: application/json" https://${wallet.walletListen}/api/v1/wallets \
+      -d '{
+      "operation": "restore",
+      "backupPhrase": ["session", "ring", "phone", "arrange", "notice", "gap", "media", "olympic", "water", "road", "spider", "rate"],
```

Comment:
Hopefully people won't try to use this one for production wallets 😉 

(I always feel like we should have some mechanism for preventing mainnet to accept any of the mnemonics we use in tests etc).

Reply:
Haha, I thought the same things :no_mouth: ... I hope it's hidden enough in the nix entrails that no one will dig this out... right ? :fearful: